app/test_split_integer.py

```diff
@@ -2,20 +2,23 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(45454, 95)) == 45454
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert len(set(split_integer(256, 4))) == 1
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(6464, 1)[0] == 6464
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(844, 93) == sorted(split_integer(844, 93))
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert split_integer(25, 32) == [
+        0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1,
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+        1, 1, 1, 1, 1, 1, 1, 1]
```

Comment:
As a general recommendation, there is no need to use large numbers in test cases when it is not a necessary part of the test case itself. It would be appropriate to shorten the compared list and the magnitude of numbers overall for this task.